src/inference/mh.jl

```diff
@@ -374,7 +374,9 @@ function propose!!(
     prev_trans = AMH.Transition(vt, getlogp(vi))
 
     # Make a new transition.
-    densitymodel = AMH.DensityModel(Turing.LogDensityFunction(vi, model, spl, DynamicPPL.DefaultContext()))
+    densitymodel = AMH.DensityModel(
```

Comment:
Isn't it easier to use a `LogDensityModel`? Then you don't need Fix1.

Reply:
Oh true!

Reply:
Wait, does that work though? AFAIK we don't support wrapping something that implements `LogDensityModel` in `AMH.DensityModel`?

Reply:
We don't want to wrap it in a `DensityModel`, do we? `step` seem to support `LogDensityModel`s: https://github.com/TuringLang/AdvancedMH.jl/blob/1638f068261f936141404e232dd7f099b2cdde95/src/mh-core.jl#L74-L113